app/token_pairs.go

```diff
@@ -0,0 +1,17 @@
+package app
+
+import erc20types "github.com/cosmos/evm/x/erc20/types"
+
+// WTokenContractMainnet is the WrappedToken contract address for mainnet
+const WTokenContractMainnet = "0xD4949664cD82660AaE99bEdc034a0deA8A0bd517"
```

Comment:
can you please add a comment where this address come from?

Reply:
I believe this is arbitrary since we are launching the EVM without the previous existence of a WETH contract. By upgrade time we could choose this and instantiate the precompile for it.